lib/shopify_cli.rb

```diff
@@ -158,3 +158,5 @@ def self.debug_log_file
     File.join(tool_config_path, 'logs', 'debug.log')
   end
 end
+
+autoload(:Result, 'result')
```

Comment:
Why in the global namespace? I would prefer it in the ShopifyCLI namespace because it makes it more clear that this is tooling that the CLI is providing and not some ruby stdlib magic.

Reply:
I'm happy to move it into the CLI namespace.